src/amp-story-player-impl.js

```diff
@@ -14,19 +14,18 @@
  * limitations under the License.
  */
 
-import {IframePool} from './amp-story-player-iframe-pool';
```

Comment:
I'd really rather see all of this in a separate folder. Can we move?

Reply:
Yes, I'll move all of the player code to a separate directory @dvoytenko